test/unit/vl-map-search.test.html

```diff
@@ -89,6 +89,38 @@
         });
       });
 
+      test('wanneer de gebruiker een Lambert coördinaat zoekt zullen bij geopunt de locaties opgehaald worden', () => {
+        const value = '104719.27, 192387.25';
+
+        const response = new Promise((resolve, reject) => {
+          resolve(new Response(JSON.stringify({}), {status: 200}));
+        });
+
+        const fetchStub = sandbox.stub(window, 'fetch');
+        const xy = value.replace(/\s/g, '');
+        fetchStub.withArgs('https://loc.geopunt.be/v4/Location?xy=' + xy + '&c=5').returns(response);
+
+        const element = fixture('vl-map-with-search-fixture');
+        return customElements.whenDefined('vl-map').then(() => {
+          const selectElement = element.shadowRoot.querySelector('vl-map-search').shadowRoot.querySelector('select');
+
+          selectElement.dispatchEvent(new CustomEvent('search', {
+            detail: {
+              value: value,
+            },
+          }));
+
+          return new Promise((resolve) => {
+            (async () => {
+              while (!fetchStub.called) {
+                await new Promise((resolve) => setTimeout(resolve, 100));
+              }
+              resolve();
+            })();
```

Comment:
ik denk dat er in vl-ui-core een awaitUntil zit of iets dergelijks.

Reply:
hmm, moet ik eens bekijken. Had deze manier overgenomen omdat die al aanwezig was in de bestaande test cases.